packages/iris-grid/src/sidebar/conditional-formatting/ConditionalFormattingUtils.ts

```diff
@@ -717,9 +717,25 @@ export function isDateConditionValid(condition: DateCondition, value?: string) {
     case DateCondition.IS_NOT_NULL:
       return true;
 
-    default:
-      // Proper date validation will be addressed by Issue #1108
-      return value != null && value !== '';
+    default: {
+      const [dateTimeString, tzCode] = (value ?? '').split(' ');
```

Comment:
Would `2023-02-23T11:46:31.000000000 NY FOO` pass validation but still fail server size?

Reply:
Good catch.
Also, a dateTime string can have a space in place of the t-separator: `2023-02-23 11:46:31.000000000 NY`.
Regex for the separator in `DateUtils/parseDateTimeString` looks like this: `[tT\s]`.

Reply:
Ignore my comment, looks like the engine expects a `T` in format conditions.

Reply:
It would be nice if we could accepted strings anywhere in the UI that takes a datetime with or without the T separator, then normalize it before sending.